README.md

```diff
@@ -35,195 +26,99 @@ https://simplereport.gov/
 
-```
-simple-report:
-  demo-users:
-    site-admin-emails:
-      - bob@example.com
-```
+- `ADMIN` - an organization admin with full access to their organization
+- `USER` - a site user the has access to everything in their organization but the gear icon
+- `ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
+- `ALL_FACILITIES` - a site user that can access all facilities in their organization
+- `NO_ACCESS` - a member of an organization who has no permissions without possessing other roles
 
-Site admins can access the `/admin` paths and site admin APIs
+These roles are controlled via Okta groups.
 
-### Restart & Clean
+## Database cleaning
 
 When there are DB schema changes the backend may throw an error and fail to start.
 
 Restarting the docker way:
 
-1. run `cd backend`
-1. Bring down the service by running `docker-compose down`
-1. Wipe the db by running `docker system prune && docker images prune && docker volume prune`
-1. Restart the service `docker-compose up --build`
+1. `docker system prune --volumes`
 
 Restarting the SQL way:
 
-1. run `db-setup/nuke-db.sh`
-2. restart the spring app `gradle bootRun --args='--spring.profiles.active=dev'`
+1. `yarn db` to open up a psql prompt
+1. `DROP DATABASE simple_report; CREATE DATABASE simple_report`
+1. Restart docker-compose
 
-### Rollbacks
+## Rollbacks
 
 The application uses the Liquibase plugin for Gradle to perform certain database management tasks.
 
 To roll the database back to its state at a prior date:
 
 ```
-$ ./gradlew liquibaseRollbackToDate -PliquibaseCommandValue=${date}
+docker-compose exec backend ./gradlew liquibaseRollbackToDate -PliquibaseCommandValue=${date}
 ```
 
 To roll back a certain _number_ of migrations:
 
 ```
-$ ./gradlew liquibaseRollbackCount -PliquibaseCommandValue=${n}
+docker-compose exec backend ./gradlew liquibaseRollbackCount -PliquibaseCommandValue=${n}
 ```
 
 To roll back to a certain tag:
 
 ```
-$ ./gradlew liquibaseUpdateToTag -PliquibaseCommandValue=${TAG}
+docker-compose exec backend ./gradlew liquibaseUpdateToTag -PliquibaseCommandValue=${TAG}
 ```
 
 If you are required to roll back a non-local database, you may generate the required SQL to execute elsewhere. Use `liquibaseRollbackToDateSQL` or `liquibaseRollbackCountSQL` in the manner described above to write the rollback SQL to stdout.
 
-### API Testing
+## API testing
 
-Go to `localhost:8080` to see interact with the graphql api. You would need to point the api endpoint to the backend at: `http://localhost:8080/graphql` This gives you a preview to query/mutate the local database.
+You can make requests to the GraphQL API using [Insomnia](https://insomnia.rest/download) or a similar client. You would need to point the api endpoint to the backend at: `https://localhost.simplereport.gov/api/graphql` This gives you a preview to query/mutate the local database.
```

Comment:
When using Insomnia, you'll also need to pass your bearer_token in the header - this can be found in Application -> Local Storage -> Access Token on a Chrome browser.